.gitignore

```diff
@@ -9,3 +9,6 @@ build*/
 /.cache
 /.tox
 __pycache__
+
+# ignore CMake generated files
```

Comment:
CMake is innocent here. This is Qt Creator's doing.

Reply:
True. Updated the comment, thanks!